pipelines/promote.py

```
@@ -420,6 +420,15 @@ def _iter_jsonl(raw_run):
         yield path, path.relative_to(raw_run)
 
 
+def _literal_lf_lines(path):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_literal_lf_lines`

(ANN202)